code/game/objects/items/devices/modificationkit.dm

```diff
@@ -0,0 +1,23 @@
+/obj/item/modkit
```

Comment:
this could probably have a better item path, its a very general path for a very specific item